src/ontology/mondo-edit.obo

```diff
@@ -499831,20 +499831,23 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/9461/autoimm
-synonym: "APD" RELATED ABBREVIATION [GARD:0009139]
-synonym: "autoimmune progesterone dermatitis" EXACT []
-synonym: "autoimmune progesterone dermatitis/urticaria" EXACT []
-synonym: "autoimmune progesterone urticaria" RELATED []
-synonym: "progesterone dermatitis" RELATED []
+name: progestogen hypersensitivity
+def: "Progestogen hypersensitivity (PH), also referred to as autoimmune progesterone dermatitis (APD), is a rare immediate or delayed hypersensitivity reaction to endogenous progesterone and/or synthetic progestins characterized by diverse cutaneous manifestations, bronchospasm, and/or anaphylaxis." [PMID:30712576, PMID:37579875]
+comment: The term “autoimmune progesterone dermatitis" was initially used because the patient described reacted to endogenous progesterone. However, there is limited evidence that this is an autoimmune condition. There is also evidence that PH may start after allergic sensitization to progestins. Thus the term “progestogen hypersensitivity” was recently proposed as an alternative to APD, as it encompasses hypersensitivity reactions to both endogenous and exogenous progesterone, as well as progestins which are closely structurally related. PMID:30712576
+synonym: "APD" RELATED ABBREVIATION [GARD:0009139, PMID:30712576]
+synonym: "autoimmune progesterone dermatitis" RELATED [PMID:30712576]
+synonym: "autoimmune progesterone dermatitis/urticaria" RELATED [MEDGEN:220359]
+synonym: "autoimmune progesterone urticaria" RELATED [MEDGEN:220359]
+synonym: "PH" EXACT ABBREVIATION [PMID:30712576, PMID:37579875]
+synonym: "progesterone dermatitis" RELATED [MEDGEN:220359]
 xref: ICD9:708.8 {source="MONDO:relatedTo", source="MONDO:i2s"}
-xref: MEDGEN:220359 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: MESH:C535299 {source="MONDO:equivalentTo"}
-xref: SCTID:400009001 {source="MONDO:equivalentTo"}
-xref: UMLS:C1260879 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:220359"}
-is_a: MONDO:0002406 {source="https://orcid.org/0000-0001-5208-3432"} ! dermatitis
-is_a: MONDO:0007179 {source="MESH:C535299", source="MONDO:Redundant"} ! autoimmune disease
+xref: MEDGEN:220359 {source="MONDO:relatedTo", source="MONDO:MEDGEN"}
+xref: MESH:C535299 {source="MONDO:relatedTo"}
+xref: SCTID:400009001 {source="MONDO:relatedTo"}
+xref: UMLS:C1260879 {source="MONDO:relatedTo", source="MONDO:MEDGEN", source="MEDGEN:220359"}
+is_a: MONDO:0000605 {source="PMID:37579875"} ! hypersensitivity reaction disease
```

Comment:
@katiermullen Why did you make these x-ref "MONDO:relatedTo"?

Reply:
My understanding with this issue is that "everyone" got it sort of not quite right. I think keeping these are equivalent would make more sense. Also, MedGen and UMLS will be automatically updated. 
I suggest that we keep them equivalent.